src/SDKs/ServiceBus/data-plane/Directory.Build.props

```diff
@@ -28,4 +22,14 @@
     -->
     <DefineConstants Condition="'$(OS)' == 'Windows_NT'">$(DefineConstants);WINDOWS</DefineConstants>
   </PropertyGroup>
+
+  <!-- 
+    For Track 1, override the engineering system signing key and use the existing key for
+    Azure messaging client libraries.
+  -->
+  <PropertyGroup Condition="'$(SignAssembly)' != 'false'">
+    <SignAssembly>true</SignAssembly>
```

Comment:
I suspect you don't need to set the SignAssembly property since you did get into this condition. You could switch the condition to be SignAssembly=true if you want to be sure.